ArduCopter/takeoff.cpp

```diff
@@ -52,7 +52,7 @@ void Mode::_TakeOff::start_m(float alt_m)
 {
     // initialise takeoff state
     _running = true;
-    take_off_start_alt_m = copter.pos_control->get_pos_desired_U_m();
+    take_off_start_alt_m = copter.pos_control->get_pos_estimate_U_m();
```

Comment:
this change from using the target (aka "desired") to using the estimate is intentional?  If "yes" then I guess it's a functional change so we should think about when these could be different and test it